pkgs/development/python-modules/combinadics/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib
+  , buildPythonPackage
+  , fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "combinadics";
+  version = "0.0.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-CFbtDgcbrFKEYknegVRSUZbc+jS0OCGN53ZYBAUAFD4=";
+  };
+  doCheck = false;
+  meta = with lib; {
+    homepage = "https://github.com/neoglez/combinadics";
+    description = "Combinatorial system of degree k ranking and unranking";
+    license = licenses.mit;
+    maintainers = with maintainers; [ ];
```

Comment:
Ideally, the package would have a maintainer (can be registered in https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) who can be contacted in case of PRs or issues regarding the packaging.